frontend/src/v5/ui/components/dashboard/dashboardProjectLayout/dashboardProjectDataFetcher.component.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ *  Copyright (C) 2024 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { TeamspacesActionsDispatchers, ProjectsActionsDispatchers } from '@/v5/services/actionsDispatchers';
+import { DashboardParams } from '@/v5/ui/routes/routes.constants';
+
+export const DashboardProjectDataFetcher = () => {
```

Comment:
This look more like a hook than a component

Reply:
Even though it doesn't return anything?

Reply:
Anyways, I'm probably ditching this design after the talk we had this morning where the NavigationBar can be passed a parameter